test/xskbench/xskbench.c

```diff
@@ -613,16 +629,16 @@ ProcessPeriodicStats(
                 (pokesRequestedDiff - pokesPerformedDiff) * 100 / pokesRequestedDiff;
         }
 
-        HRESULT res =
+        XDP_STATUS res =
             XdpApi->XskGetSockopt(Queue->sock, XSK_SOCKOPT_STATISTICS, &stats, &optSize);
         ASSERT_FRE(res == S_OK);
         ASSERT_FRE(optSize == sizeof(stats));
 
         rxDropDiff = stats.RxDropped - Queue->lastRxDropCount;
-        rxDropKpps = rxDropDiff ? (double)rxDropDiff / tickDiff : 0;
+        rxDropKpps = rxDropDiff ? rxDropDiff / tickDiff : 0;
         Queue->lastRxDropCount = stats.RxDropped;
 
-        printf("%s[%d]: %9.3f kpps %9.3f rxDropKpps rxDrop:%llu rxTrunc:%llu "
+        printf("%s[%d]: %llu kpps %llu rxDropKpps rxDrop:%llu rxTrunc:%llu "
```

Comment:
Does kernel not support nice formatting of integers? Consistent padding makes these a lot easier to read

Reply:
I was seeing issues with tracing floating point types in kernel. I might have tried a few things but couldn't figure it out quickly so this was just a quick workaround for now

Reply:
Maybe that's not too surprising, because the kernel really does not like floating point types. Maybe we can update the code to perform integer arithmetic instead of floats?

Reply:
I missed that the original comment was about the padded formatting. I've restored the padding. I briefly looked into using only integer arithmetic and realized we currently have a dependency on sqrt(double, double)